ui/client/containers/SubProcesses.js

```diff
@@ -34,49 +34,18 @@ class SubProcesses extends BaseProcesses {
       <div className="Page">
         <HealthCheck/>
         <div id="process-top-bar">
-          <div id="table-filter" className="input-group">
-            <input
-              type="text"
-              placeholder="Filter by text.."
-              className="form-control"
-              aria-describedby="basic-addon1"
-              value={this.state.search}
-              onChange={this.onSearchChange}
-            />
-            <span className="input-group-addon" id="basic-addon1">
-              <img id="search-icon" src={filterIcon}/>
-            </span>
-          </div>
+          <SearchFilter onChange={this.onSearchChange}
+                        value={this.state.search}/>
 
-          <div id="categories-filter" className="input-group">
-            <Select
-              isMulti
-              isSearchable
-              defaultValue={this.state.selectedCategories}
-              closeMenuOnSelect={false}
-              id="categories"
-              className="form-select"
-              options={this.props.filterCategories}
-              placeholder="Select categories.."
-              onChange={this.onCategoryChange}
-              styles={this.customSelectStyles}
-              theme={this.customSelectTheme}
-            />
-          </div>
-          
-          {
-            this.props.loggedUser.isWriter() ? (
-              <div
-                id="process-add-button"
-                className="big-blue-button input-group"
-                role="button"
-                onClick={() => this.setState({showAddProcess: true})}
-              >
-                CREATE NEW SUBPROCESS
-                <img id="add-icon" src={createProcessIcon}/>
-              </div>
-            ) : null
-          }
+          <TableSelect defaultValue={this.state.selectedCategories}
+                       options={this.props.filterCategories}
+                       placeholder={"Select categories.."}
+                       onChange={this.onCategoryChange}
+                       isMulti={true}
+                       isSearchable={true}/>
+
+          <AddProcessButton loggedUser={this.props.loggedUser}
```

Comment:
All attributes in one line? :)

Reply:
Why? It excedds Intelij default max width line